docs/windows_instructions.md

```diff
@@ -0,0 +1,105 @@
+# English Version
+---
+
+## Preparing to contribute with the Querido Diário’s scrapers
+
+The [Querido Diário’s scrapers][raspQD] are made on [Python 3] with [Scrapy] lib and it’s hosted on [GitHub]. This guide assume that you’re starting your contact with Python e you are already have Python 3 installed, hence we’re going to introduce you what you need to prepare yourself for the sprint: GitHub, git, dependencies set up and a guide showing you how to analyses websites with scrapers.
+##### <ins><div id="win-en">Step by step on Windows</div></ins>
+
+1. The contribution platform is GitHub, so let’s [create an account][account].
+
+2. Access the [Querido Diário’s scrapers official repository][repQD]
+
+3. Create your fork from the project. [Here][tutoGitHub] there is a contribution tutorial on GitHub (Read until step 01 “How to open a Pull Request?”).
+
+4. Let’s download and install [VSCode] on your computer (you can follow the pattern installation).
+- The VSCode isn’t mandatory, any text editor works, but to simplify, during the sprint we’ll assume that you’re using the terminal with PowerShell on VSCode.
```

Comment:
Could you please break the lines? It will help us when reviewing the text in the future.